Makefile

```diff
@@ -13,10 +13,10 @@ ANSIBLE_PLAYBOOK_CMD = ansible-playbook \
 all:
 	$(error Usage: make <aws|gce> DEPLOY_ENV=name [ARGS=extra_args])
 
-aws: preflight check-env-aws
+aws: preflight check-env-aws clean-roles ansible-galaxy
```

Comment:
We could add these to the `preflight` target to prevent the duplication in the `aws` and `gce` targets.